lib/install/web.rb

```diff
@@ -10,6 +10,20 @@ def node_version_unsupported?
   node_version < "20.0.0"
 end
 
+def rails_version
+  version = `rails --version`[/\d+\.\d+\.\d+/]
+
+  return if version.blank?
+
+  Gem::Version.new(version)
+end
+
+def rails_version_unsupported?
+  minimum_rails_version = Gem::Version.new("7.2.0")
+
+  rails_version < minimum_rails_version
+end
```

Comment:
I experimented with this locally, and we have access to [`Rails#gem_version`](https://api.rubyonrails.org/classes/Rails.html#method-c-gem_version) in the template. This is neat because it returns the [version](https://api.rubyonrails.org/classes/Rails.html#method-c-version) as an instance of `Gem::Version`.

```suggestion
def rails_version_unsupported?
  Rails.gem_version < Gem::Version.new("7.2.0")
end
```

Reply:
That's great and cleaned this up a lot! `Rails.gem_version` implemented in 62105edb3fc797c0f6ec983d50988ad4176f9b96.

Reply:
Make that 67f7a30fa6d0ec9055edce5f15bac1f5d41f0526.